### ISSUE TITLE
Create a common props file for package version consistency

### ISSUE THREAD

**Issue by m-nash** [Azure/autorest.csharp#1143]
Right now each project in the solution defines its own version for each dependency.  This can cause issues, and we should probably define a common props file to declare the version to use.  This will also save us from having to remember where to update when we want to change the version of a package we depend on.

**Comment by chamons** [comment]
Good point. We could probably just leverage a Directory.Build.Common.props in the root to set many project settings.
